custom_components/tahoma/water_heater_devices/domestic_hot_water_production.py

```diff
@@ -51,13 +74,20 @@ def max_temp(self):
     def current_operation(self):
         """Return current operation ie. eco, electric, performance, ..."""
         return OVERKIZ_TO_OPERATION_MODE[
-            self.executor.select_state(OverkizState.IO_DHW_MODE)
+            OVERKIZ_TO_OPERATION_MODE[OverkizCommandParam.BOOST]
+            if self._is_boost_mode_on
+            else self.executor.select_state(
+                OverkizState.IO_DHW_MODE, OverkizState.MODBUSLINK_DHW_MODE
+            )
         ]
 
     @property
     def current_temperature(self):
         """Return the current temperature."""
-        return self.executor.select_state(OverkizState.IO_MIDDLE_WATER_TEMPERATURE)
+        return self.executor.select_state(
+            OverkizState.IO_MIDDLE_WATER_TEMPERATURE,
+            OverkizState.MODBUSLINK_MIDDLE_WATER_TEMPERATURE,
```

Comment:
```suggestion
            OverkizState.MODBUSLINK_MIDDLE_WATER_TEMPERATURE,
```
This state isn't defined in pyoverkiz... @rguillard77 can you have a look and add the correct states to pyoverkiz or fix them in here? 
same for `MODBUSLINK_DHW_ABSENCE_MODE `and `MODBUSLINK_DHW_MODE`

Reply:
scratch that, I was not on the latest version of pyoverkiz...